### ISSUE TITLE
Fatal IO error 11 (Resource temporarily unavailable) on X server :1

### ISSUE THREAD

**Issue by sergiud** [pdfpc/pdfpc#253]
I'm on Arch Linux and GNOME 3.24. Whenever I try to switch to the second slide of `video-example.pdf`, `pdfpc` exits with the message
```
Gdk-Message: pdfpc: Fatal IO error 11 (Resource temporarily unavailable) on X server :1.
```

**Comment by phillipberndt** [comment]
Could you check whether [this PR fixes the problem](https://github.com/pdfpc/pdfpc/pull/251), please?

**Comment by sergiud** [comment]
Thanks! Replacing `glimagesink` by `glimagesinkXXX` solved the problem. The PR however did not work for me. `pdfpc` still exists with the above message.

**Comment by phillipberndt** [comment]
@AndreasBilke That's the 3rd or 4th regression introduced by #228; I'd tend to partially revert it at this point. For example, we could add a command line option or rc variable for users to explicitly opt into using the glimagesink. Or we could try to find an alternative fix for the no-sound issue, like routing the audio to a different sink.

**Comment by AndreasBilke** [comment]
I am fine with reverting this commit. Go ahead.

**Comment by AndreasBilke** [comment]
Fixed with #251
